specification/iothub/resource-manager/Microsoft.Devices/preview/2022-04-30-preview/iothub.json

```diff
@@ -2932,7 +2972,8 @@
         },
         "connectionString": {
           "description": "The connection string of the service bus queue endpoint.",
-          "type": "string"
+          "type": "string",
+          "x-ms-secret": true
```

Comment:
Does it mean the property will not be returned in response?

Reply:
I believe so, @didact2401 would you double check on this?

Reply:
I believe that is correct, yes.